docker/host.sls

```diff
@@ -53,5 +53,17 @@ docker-py:
     - reload_modules: True
 {% endif %}
 
+{%- if host.registries is defined %}
+
+{%- for registry in host.registries %}
+
+docker_{{ registry.address }}_login:
+  cmd.run:
+  - name: 'docker login -u {{ registry.user }} -p {{ registry.password }} {{ registry.address }}'
+  - unless: grep {{ registry.address }} .docker/config.json
```

Comment:
Use absolute path, `/root/.docker/config.json` if it's really going to run under root (is it?)